storage/2018-11-09/datalakestore/filesystems/resource_id.go

```diff
@@ -15,6 +15,13 @@ func (client Client) GetResourceID(accountName, shareName string) string {
 	return fmt.Sprintf("%s/%s", domain, shareName)
 }
 
+// GetResourceManagerResourceID returns the Resource Manager specific
+// ResourceID for a Data Lake Storage FileSystem
+func (client Client) GetResourceManagerResourceID(subscriptionID, resourceGroup, accountName, fileSystemName string) string {
+	fmtStr := "/subscriptions/%s/resourceGroups/%s/providers/Microsoft.Storage/storageAccounts/%s/blobServices/default/containers/%s"
```

Comment:
@tombuildsstuff this is for [Datalake Gen 2](https://docs.microsoft.com/en-us/azure/storage/blobs/data-lake-storage-introduction), so it will follow the Resource Manager ID of a Blob Storage container. Unless there's a Data Lake Gen 2 Filesystem Resource Manager ID I'm overlooking

Reply:
@tombuildsstuff is this still an issue in getting this merged?